sdk/maps/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: maps
+    BuildInParallel: true
+    TimeoutInMinutes: 180
+    Clouds: 'Public'
+    SupportedClouds: 'Public,Canary'
```

Comment:
In case this is copy pasted and you don't need tests against early access regions running on the weekends:

```suggestion
```